cbindings/wrapper.go

```diff
@@ -706,11 +720,25 @@ func (w *CWrapper) GetCollections(
 }
 
 func (w *CWrapper) GetAllIndexes(ctx context.Context) (map[client.CollectionName][]client.IndexDescription, error) {
-	colName := C.CString("")
-	defer C.free(unsafe.Pointer(colName))
+	cVersion := C.CString("")
+	cCollectionID := C.CString("")
+	cName := C.CString("")
+	cIdentity := identityFromContext(ctx)
+	defer C.free(unsafe.Pointer(cVersion))
+	defer C.free(unsafe.Pointer(cCollectionID))
+	defer C.free(unsafe.Pointer(cName))
+	defer C.IdentityFree(cIdentity)
+	defer C.IdentityFree(cIdentity)
```

Comment:
todo: There is a duplicate free.

```suggestion
	defer C.IdentityFree(cIdentity)
	defer C.IdentityFree(cIdentity)
```